firefox-ios/Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -164,6 +168,28 @@ class ClearPrivateDataTableViewController: ThemedTableViewController {
         tableView.deselectRow(at: indexPath, animated: false)
     }
 
+    private func dequeueCellFor(indexPath: IndexPath, isCenteredTableViewCell: Bool = false) -> ThemedTableViewCell {
```

Comment:
Instead of creating your own method here, it would be better to override the `dequeueCellFor(indexPath:)` and checking for the section. This way you also won't have to call `dequeueCellFor(indexPath:)` and `applyTheme(theme:)` in every if block as the overridden method will return the correct cell.

Reply:
Did the override but I should check if the dequeued cell is a specific type of cell because I have to call some custom methods for ThemedCenteredTableViewCell.